plugins/proxy-backend/src/service/router.ts

```diff
@@ -121,6 +127,9 @@ export function buildMiddleware(
 
   // Attach the logger to the proxy config
   fullConfig.logProvider = () => logger;
+  if (isValidHttpProxyMiddlewareLogLevel(logger.level)) {
```

Comment:
The ability to inspect the log level of the logger is about to disappear, so I'm hesitant to add this. Would it make sense to always pass a high log level here and then handle filtering or our end instead through the custom logger passed to the middleware?

Reply:
Sounds good. I systematically pass in `debug` now (the most verbose option).